src/Hint/Annotations.hs

```diff
@@ -18,12 +17,13 @@ getModuleAnnotations :: (Data a, MonadInterpreter m) => a -> String -> m [a]
 getModuleAnnotations _ x = do
     mods <- GHC.mgModSummaries . hsc_mod_graph <$> runGhc GHC.getSession
     let x' = filter ((==) x . GHC.moduleNameString . GHC.moduleName . ms_mod) mods
-    concat <$> traverse (anns . ModuleTarget . ms_mod) x'
+    concatMapM (anns . ModuleTarget . ms_mod) x'
 
 -- Get the annotations associated with a particular function.
 getValAnnotations :: (Data a, MonadInterpreter m) => a -> String -> m [a]
-getValAnnotations _ =
-    fmap concat <<< traverse (anns . NamedTarget) <=< runGhc1 GHC.parseName
```

Comment:
I find `f <<< g <=< h` to be especially unreadable, because `(f <<< g) <=< h` and `f <<< (g <=< h)` are different expressions!